### ISSUE TITLE
Document that all links should have underlines in the Frontend Playbook

### ISSUE THREAD

**Issue by Heydon** [springernature/frontend-playbook#436]
[] Add note to playbook
[] Cite WCAG 1.4.1
[] Open to review by people who attended the open-space discussion

**Comment by jpw** [comment]
Was done, thank you!
https://github.com/springernature/frontend-playbook/commit/acf727e461df69e36714f9ec869fc4ac97f9c78c
